petfinder/client.py

```diff
@@ -179,7 +179,7 @@ def _parse_pet_record(self, root):
             # copy to the record as key/val.
             node = root.find(field)
             if node is None:
-                print "SKIPPING %s" % field
+                print ("SKIPPING %s") % field
```

Comment:
Looks like there's a space between print and the opening parenthesis. I'd also prefer that the substitution happen within the parens:
```
print ("SKIPPING %s" % field )
```